server/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -410,3 +410,22 @@ SqlDrop SqlDropFunction(Span s, boolean replace) :
         return SqlDdlNodes.dropFunction(s.end(this), ifExists, id);
     }
 }
+
```

Comment:
Same response as above - this is done the same way as in the SqlCreate function. I am not totally clear on what the benefit of having SqlDdlNodes here is either, but I think it makes sense to stay consistent with how the existing code handles similar functionalities.

Reply:
I'm not sure I'm following this comment thread.

@ArthurInTheShell do you mean wrap the definition of SqlReplac() in a #if? If so, since it's defined in babel's parserImpls.ftl file, we can omit that.

@dstekol SqlDdlNodes is the name of the class where dropFunction is defined. Can you clarify on what you mean? Is there an alternative to not using its dropFunction method?

Reply:
Actually, I think I might be referring to the wrong part of the code. SqlDdlNodes has methods that are used (elsewhere in the code) to create the objects (rather than directly calling the constructor). For example, there is a createView method in SqlDdlNodes that returns a new SqlCreateView object by calling the constructor. I'm not sure why calling this method is better than calling the constructor directly, but this is the pattern that has been used, so I've been trying to stick to it.
@ArthurInTheShell could you clarify what you had in mind?

Reply:
Drew's understanding of my advice is correct, I think we can omit that since it is in babel/parserImpls.ftl . Btw, I think we should keep the comment as unresolved if it is still open. I saw notification from github saying someone mentioned me in a comment but I can't find where exactly I am being mentioned.

Reply:
Ok, sounds good. You're right, I may have been too trigger-happy with the resolve button.